jsapp/scss/components/_kobo.modal.scss

```diff
@@ -484,6 +484,281 @@
   }
 }
 
+.form-modal--translation-settings {
+  .form-view__cell--translation-note {
+    padding-bottom: 20px;
+  }
+  .form-view__cell--translation {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-top: 1px solid $divider-color;
+    padding: 6px 0px;
+  }
+
+  .form-view__cell--translation-actions {
+    min-width: 150px;
+    text-align: right;
+  }
+
+  .form-view__cell--add-language {
+    padding-top: 20px;
+    text-align: right;
+    border-top: 1px solid $divider-color;
+  }
+
+  .form-view__cell--add-language-form,
+  .form-view__cell--update-language-form {
+    background: #F8F8F8;
+    padding: 15px;
+    position: relative;
+
+    .form-view__link--close {
+      position: absolute;
+      right: 0px;
+      top: 6px;
+
+      i {
+        font-size: 24px;
+      }
+    }
+  }
+
+  .form-view__cell--add-language-form {
+    margin-top: 20px;
+  }
+
+  .form-view__cell--add-language-fields {
+    display: flex;
+    justify-content: space-between;
+
+    .form-view__cell {
+      min-width: 100px;
+
+      &--lang-code {
+        max-width: 50px;
+      }
+
+      input {
+        padding-left: 5px;
+        max-width: 95%;
+      }
+
+      button {
+        margin-left: 10px;
+        margin-top: 5px;
+      }
+    }
+  }
+}
+
+.form-modal--translation-table {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin: 0;
+
+  .form-modal__item--translation-table--container {
+    height: calc(100% - 80px);
+    overflow: auto;
+    border: 1px solid $divider-color;
+  }
+
+  .ReactTable {
+    width: 100%;
+
+    .rt-thead.-header {
+      box-shadow: none;
+      border-bottom: 2px solid $divider-color;
+    }
+
+    .rt-thead > .rt-tr,
+    .rt-tr-group > .rt-tr {
+      .rt-td {
+        white-space: normal;
+      }
+      > .rt-th:first-child,
+      > .rt-td:first-child {
+        min-width: 30%;
+        color: lighten($cool-gray, 10%);
+      }
+
+      > .rt-th:last-child,
+      > .rt-td:last-child {
+        min-width: 70%;
+      }
+      .translation {
+        background: lighten($cool-green, 41%);
+        color: lighten($black, 15%);
+        padding: 0px;
+        textarea {
+          padding: 8px;
+          border: none;
+          background: transparent;
+          &:focus {
+            background: lighten($cool-green, 35%);
+          }
+        }
+      }
+    }
+  }
+
+  .form-modal__item--translation-table--actions {
+    width: 100%;
+    text-align: right;
+    padding-top: 20px;
+    height: 80px;
+  }
+}
+
+// alertify overrides
```

Comment:
all those styles below should be in `jsapp/scss/components/_kobo.alertify-overrides.scss` (from quick look they seem to be 100% duplicated there, but better diffcheck it)